src/index.html

```diff
@@ -8,6 +8,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star stars__star--rm-margin"></div>
```

Comment:
```suggestion
      <div class="stars__star"></div>
```
This class is redundant. Consider using the pseudoclass in this case